docs/usage/projectswift.mdx

```diff
@@ -1322,3 +1345,26 @@ A path represents a path to a file, directory, or a group of files represented b
   title="ExpressibleByStringLiteral"
   description="The path can be initialized with a plain string. It's equivalent to using `.relativeToManifest.` except when the path starts with `//`, in which case the path is considered relative to the root directory."
 />
+
+## Target Reference
+
+<PropertiesTable
+  properties={[
+    {
+      name: 'path',
+      description:
+        'Path to the target's project.',
```

Comment:
Nitpick: To the project or the directory that contains the project?

Reply:
Good catch